release-1.8/release_notes_draft.md

```diff
@@ -66,6 +66,7 @@ fundamental aspect of a secure cluster.
 
 * Advanced auditing has graduated from `v1alpha1` to `v1beta1` with the
   following changes to the default behavior.
+  * Advanced auditing feature gate is enabled by default
```

Comment:
Feature gate for advanced auditing is enabled by default.

 or

 Advanced auditing is enabled by default.

(suggesting to remove the long noun string)

Reply:
Done.